app/src/main/kotlin/com/wire/android/ui/home/conversations/details/dialog/ClearConversationContentDialog.kt

```diff
@@ -0,0 +1,51 @@
+package com.wire.android.ui.home.conversations.details.dialog
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.res.stringResource
+import com.wire.android.R
+import com.wire.android.ui.common.VisibilityStateExt
+import com.wire.android.ui.common.WireDialog
+import com.wire.android.ui.common.WireDialogButtonProperties
+import com.wire.android.ui.common.WireDialogButtonType
+import com.wire.android.ui.common.button.WireButtonState
+import com.wire.android.ui.common.visbility.VisibilityState
+import com.wire.android.ui.home.conversationslist.model.DialogState
+
+@Composable
+fun ClearConversationContentDialog(
+    dialogState: VisibilityState<DialogState>,
+    isLoading: Boolean,
+    onClearConversationContent: (DialogState) -> Unit
+) {
+    LaunchedEffect(isLoading) {
+        if (!isLoading) {
+            dialogState.dismiss()
+        }
+    }
+
+    VisibilityStateExt(dialogState) {
+        WireDialog(
+            title = "Clear content?",
+            text = "This will clear the previous conversation history on all your devices. You remain in the ${it.conversationTypeDetail.label} and have access to all new ${it.conversationTypeDetail.label} activity.",
```

Comment:
hardcoded texts???????????

Reply:
you resolve the discussion, but hardcoded text is still there

Reply:
Make sure this is extract to proper string keys @trOnk12